content/compatibility/_index.md

```diff
@@ -6,11 +6,7 @@ description: "Compatible Implementations of Jakarta EE"
 hide_sidebar: true
 draft: false
 links: [[href: "get_listed", text: "Get Listed"]]
+layout: "single"
 ---
-<div class="row">
-	<div class="col-md-16">
-	The success of Jakarta EE technology relies on compatibility to ensure application portability, security, stability, and resiliency. Compatibility will allow more technology providers to leverage and strengthen the Jakarta EE brand in the enterprise Java community. If you are interested in featuring your product here, follow the steps on the [Get Listed](get_listed) page.
-	</div>
-  <div class="col-md-8"><img class="compatible-logo img img-responsive" src="/images/jakarta/jakarta-ee-compatible-logo-color.svg" title="Jakarta EE logo"></div>
-</div>
-{{% compatible_products %}}
+
+{{< compatible-products-page >}}
```

Comment:
Maybe we should rename this shortcode to page-compatible-products and rename the other shortcodes of the same category by putting "page-" in front. This will list all the shortcodes by categories in the directory.

Reply:
+1 As discussed with you, I will create a pages folder for these shortcode.